packages/perseus-editor/src/components/__stories__/color-select.stories.tsx

```diff
@@ -1,8 +1,9 @@
+import {type LockedFigureColor} from "@khanacademy/perseus";
```

Comment:
```suggestion
import {LockedFigureColor} from "@khanacademy/perseus";
```
Does it work like this? Do we still need `type`?

Reply:
Yeah, it looks like I do need `type`